tracking.py

```diff
@@ -130,25 +143,25 @@ def receive_buffer(tweets):
 
 
 def __tool_logo():
-    print(colored('''\n\t\t\033[1m        
-             ___________       .__.__  __                                          
-             \__    ___/_  _  _|__|__|/  |_  ___________                           
-               |    |  \ \/ \/ /  |  \   __\/ __ \_  __ \                          
-               |    |   \     /|  |  ||  | \  ___/|  | \/                          
-               |____|    \/\_/ |__|__||__|  \___  >__|                             
-            .___        __         .__  .__  .__\/                                 
-            |   | _____/  |_  ____ |  | |  | |__| ____   ____   ____   ____  ____  
-            |   |/    \   __\/ __ \|  | |  | |  |/ ___\_/ __ \ /    \_/ ___\/ __ \ 
-            |   |   |  \  | \  ___/|  |_|  |_|  / /_/  >  ___/|   |  \  \__\  ___/ 
+    print(colored('''\n\t\t\033[1m
+             ___________       .__.__  __
+             \__    ___/_  _  _|__|__|/  |_  ___________
```

Comment:
Tool Logo is incorrected format when running tracking.py. You can look https://ibb.co/cN0hfV link.

Reply:
Ah let me check that out.

Reply:
Fixed